website/static/js/addProjectPlugin.js

```diff
@@ -144,6 +153,54 @@ var AddProject = {
                                 name : 'projectName'
                             })
                         ]),
+                        window.contextVars.currentUser.institutions.length ? m('.form-group.m-v-sm', [
+                            m('label.f-w-lg.text-bigger', 'Affiliation'),
+                            m('a', {onclick: function(){
+                                $('.img-circle.fa.fa-check').addClass('hidden');
+                                ctrl.newProjectInstitutions = [];
+                            }, style: {float: 'right'}},'Remove all'),
+                            m('a', {onclick: function(){
+                                $('.img-circle.fa.fa-check').removeClass('hidden');
+                                ctrl.newProjectInstitutions = window.contextVars.currentUser.institutions.map(
+                                    function(inst){return inst.id;}
+                                );
+                            }, style: {float: 'right', marginRight: '5px'}}, 'Select all'),
+                            m('table', m('tr', window.contextVars.currentUser.institutions.map(
+                                function(inst){
+                                    return m('td',
+                                        m('a', {onclick: function(){
+                                            $('#instLogo' + inst.id).toggleClass('hidden');
```

Comment:
This dependence on CSS IDs and manual DOM updates should be unnecessary. Consider refactoring (store state on ctrl and update view accordingly...).